composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "flarum/core",
+    "name": "zhishiq/flarum-core",
```

Comment:
Don't change this please.

Reply:
It's weird, I didn't include this change to the pull request...